taccsite_cms/static/site_cms/css/src/_imports/elements/html-elements.css

```diff
@@ -28,7 +28,12 @@ body {
 }
 
 body {
+  /* To avoid negative whitespace at right on horz scroll on tiny screen */
+  min-width: 290px; /* developer-decided value */
+
   font-family: var(--global-font-family);
+
+  line-height: 1.4; /* overwrite Bootstrap */
```

Comment:
In Frontera, this is a noop. In other CMS's, this means line-height changes from `1.5` to `1.4` (negligible space between lines of text). This value came from the [Frontera Homepage Redesign](https://github.com/TACC/Core-CMS/issues/157), which—on average—has a 1.4 line-height.